bokehjs/src/lib/models/ui/tooltip.ts

```diff
@@ -387,6 +396,8 @@ export class TooltipView extends UIElementView {
 
     this.el.style.top = `${top}px`
     this.el.style.left = `${left}px`
+
+    this.update_bbox()
```

Comment:
Just for my own understanding, why do these changes require you to update the bounding box now but not previously?

Reply:
It was always needed, because `update_bbox()` is generally badly implemented and this PR and a few other show this even more. Eventually I will have to redesign this, but for now it's better than having random numbers in `*.blf` files.